core/src/eval/mod.rs

```diff
@@ -124,7 +124,7 @@ pub struct VirtualMachine<R: ImportResolver, C: Cache> {
     // The call stack, for error reporting.
     call_stack: CallStack,
     // The interface used to fetch imports.
-    import_resolver: R,
+    pub import_resolver: R,
```

Comment:
I think maybe it was originally for borrow-check reasons (because `import_resolver_mut()` borrows the whole VirtualMachine), but it isn't needed anymore.